application/__tests__/example.macro.integration.js

```diff
@@ -2,12 +2,23 @@
 
 const { integrationTestPort } = require('../../constants')
 
-describe('Localhost', () => {
+describe('Example page rendering', () => {
   beforeEach(async () => {
-    await page.goto(`http://localhost:${integrationTestPort}`)
+    await page.goto(`http://localhost:${integrationTestPort}/design-library/ask-for-consent/`)
   })
 
-  test('should be titled "HMRC Design System"', async () => {
-    await expect(page.title()).resolves.toMatch('HMRC Design System')
+  test('should be titled "Ask the user for their consent"', async () => {
+    const title = await page.title()
+    expect(title).toMatch('Ask the user for their consent')
+  })
+
+  test('iframes should have a style attribute with a height property', async () => {
+    const noHeightProperty = await page.$$eval('iframe', iframes => iframes.filter(iframe => !iframe.style.height))
+    expect(noHeightProperty).toHaveLength(0)
+  })
+
+  test('iframes should be resized to display all content', async () => {
+    const resizeFailed = await page.$$eval('iframe', iframes => iframes.filter(iframe => iframe.offsetHeight < iframe.contentWindow.document.body.offsetHeight))
```

Comment:
I'd put this into a function, just to give it a name:

```
iframes.filter(isFrameATardis)
```

You'd want a better name than that, it's a function which returns true if the frame is bigger on the inside so we could call it `isFrameContentBiggerThanFrameItself` etc.

Reply:
👍 Tardis name retained due to outstanding clarity